HACKING.md

```diff
@@ -288,13 +288,16 @@ To make a release of the ops library, do the following:
    link to the full changelog.
 8. Change [version.py][ops/version.py]'s `version` to the
    [appropriate string](https://semver.org/).
-9. Add, commit, and push, and open a PR to get the changelog and version bump
+9. Update the pinned charm versions in `.github/workflows/*-charm-tests.yaml` to
```

Comment:
I like when these are links, although it's trickier here since it's a wildcard. Maybe to the directory?

```suggestion
9. Update the pinned charm versions in [*-charm-tests.yaml](.github/workflows/) to
```